mp/src/game/shared/momentum/mom_explosive.cpp

```diff
@@ -11,6 +12,10 @@
 
 #include "tier0/memdbgon.h"
 
+MAKE_TOGGLE_CONVAR(mom_explosive_sound_fizzle_enable, "1", FCVAR_ARCHIVE | FCVAR_REPLICATED, "Toggles the explosive fizzle sound. 0 = OFF, 1 = ON\n");
+
+#define SND_EXPLOSIVE_FIZZLE_DEFAULT "Weapon_MegaPhysCannon.DryFire"
+
```

Comment:
Should be an entry in the SoundData block in the weapon of choice. PlayFizzleSound can then be overridden between all 3 explosives to fetch and play the sound from the weapon data.

This would handle precaching the sound automatically as well, so Precache can be removed.

You can then also add a new `mom_<explosivetype>_sound_fizzle_enable` for each of them, to match the other commands for `mom_rj_sound_*` and `mom_sj_sound_*`.

Reply:
I did for rocket/sticky as that's all i see on `CMomExplosive`. I assume the 3rd explosive you mean conc?

Reply:
Yeah, the conc grenade will be the third